mentoring/PROJECT_IDEA_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+# CCC Mentorship Program: Project Idea Template
+
+Welcome to the Confidential Computing Consortium (CCC) Mentorship Program. This guide is designed to help sponsors submit project ideas that will both challenge and nurture aspiring contributors interested in confidential computing. Below, you'll find a template to help you craft a comprehensive project proposal, featuring an example based on a real project idea for Enarx.
+
+## Project Idea Submission Template
+
+### Template
+
+#### CCC Project Name
+##### Title
+
+- **Description**:
+- **Expected Outcome**:
+- **Recommended Skills**:
+- **Mentor(s)**: # It is recommended to have at least 2 mentors, and at least one of them should be the primary mentor. For GSoC, it is **required** to have at least 2 mentors.
```

Comment:
```suggestion
- **Mentor(s)**: # It is recommended to have at least 2 mentors, and at least one of them should be the primary mentor.
```
Recommend dropping mention of the GSoC requirements here to avoid confusion with our requirements. At least I think that GSoC is just mentioned in these guides as a reference for how to mentor and is not a program that the CCC is affiliated with.

Reply:
Updated in my commit.